vllm/entrypoints/llm.py

```diff
@@ -277,15 +277,15 @@ def generate(
     def generate(
         self,
         prompts: Union[Union[PromptInputs, Sequence[PromptInputs]],
-                       Optional[Union[str, List[str]]]] = None,
+                       Optional[Union[str, List[str]]], ] = None,
```

Comment:
Please avoid formatting changes in sections of the code that are unrelated to this PR.

Reply:
I think `./format.sh` makes these changes? Should I not run the ruff linter + formatter

Reply:
`format.sh` should only run `ruff check` and format using `yapf`. It doesn't make those changes on my end.

Reply:
Make sure your IDE doesn't have auto-formatters enabled.